indico/modules/networks/templates/delete_network.html

```diff
@@ -0,0 +1,27 @@
+{% from 'confirmation_dialog.html' import confirmation_dialog %}
+
+{% set ok_text=_("Delete network group") %}
+
+{% if in_use %}
+    {% call confirmation_dialog('danger', message=_("Deleting the network will remove the ACL entries referencing it. This may lead to unwanted protection settings for the categories and events mentionned above."), ok_text=ok_text) %}
```

Comment:
I think setting the message via `{% set %}` would be more readable. Actually, I think you can even do this:
```
{% set confirmation_message %}
    {% trans %}..{% endtrans %}
{% endset %}
```

might be more readable since the message is somewhat long